testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/beanvalidation/beanvalidationtest/JPABeanValidationTestCase.java

```diff
@@ -44,6 +45,7 @@ public static Archive<?> deploy() {
                 SFSB1.class
         );
         jar.addAsManifestResource(JPABeanValidationTestCase.class.getPackage(), "persistence.xml", "persistence.xml");
+        jar.addAsManifestResource(new StringAsset("<beans bean-discovery-mode=\"none\"></beans>"), "beans.xml");
```

Comment:
Why do we need this? Is CDI expected to not be present for this test?

Reply:
We could remove my commit to add these changes if it helps our process.  I added those changes to ensure a few different cases are checked but could remove them.

Reply:
I removed these test changes just in case they have anything to do with why the tests are now passing but didn't seem to before.